starknet_devnet/state.py

```diff
@@ -0,0 +1,29 @@
+"""
+Global state singletone
+"""
+
+from .starknet_wrapper import StarknetWrapper
+from .dump import Dumper
+
+class State():
+    """
+    Stores starknet wrapper and dumper
+    """
+    def __init__(self):
+        self.starknet_wrapper = StarknetWrapper()
+        self.dumper = Dumper(self.starknet_wrapper)
+
+    def set_starknet_wrapper(self, starknet_wrapper: StarknetWrapper):
```

Comment:
Is this used anywhere outside this class? If not, you can add a double underscore in the beginning of the name to make it private.

Reply:
[5fd05f3](https://github.com/Shard-Labs/starknet-devnet/pull/64/commits/5fd05f3c911372162540c0d5833275d8bad5c018)